lambda/writePass/index.js

```diff
@@ -252,95 +251,112 @@ exports.handler = async (event, context) => {
 
         const existingItems = await dynamodb.query(existingPassCheckObject).promise();
 
-        if (existingItems.Count > 0) {
-          return sendResponse(400, {
-            title: 'This email account already has a reservation for this booking time.',
-            msg: 'A reservation associated with this email for this booking time already exists. Please check to see if you already have a reservation for this time. If you do not have an email confirmation of your reservation please contact <a href="mailto:parkinfo@gov.bc.ca">parkinfo@gov.bc.ca</a>'
-          });
-        }
+        // if (existingItems.Count > 0) {
+        //   return sendResponse(400, {
+        //     title: 'This email account already has a reservation for this booking time.',
+        //     msg: 'A reservation associated with this email for this booking time already exists. Please check to see if you already have a reservation for this time. If you do not have an email confirmation of your reservation please contact <a href="mailto:parkinfo@gov.bc.ca">parkinfo@gov.bc.ca</a>'
+        //   });
+        // }
```

Comment:
Was this committed by accident (i.e. commented out for dev purposes)?

Reply:
f344c2c6c08a8ec9838b9b2e68015f8b002450b6